### PR TITLE
[fix]: fix mlu_ops docs

### PR DESCRIPTION
**Pull request by tudejiang79**
Thanks for your contribution and we appreciate it a lot. Please state your motivation and modification for this pull_request here.

## 1. Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## 2. Modification

Please briefly describe what modification is made in this PR ，and indicate where to modify it.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -7,3 +7,79 @@ Please describe the motivation of this PR and the goal you want to achieve throu
 ## 2. Modification
 
 Please briefly describe what modification is made in this PR ，and indicate where to modify it.
+
+## 3. Test Report
+
+If you want to know how to do operator testing, you can see [GTest-User-Guide-zh](../docs/GTest-User-Guide-zh.md)
+
+### 3.1 Modify The Description
```
Comment (defei-coder) [inline .github/pull_request_template.md:15]: 修改描述？Or The Description of modification?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -7,3 +7,79 @@ Please describe the motivation of this PR and the goal you want to achieve throu
...
+|        1       |          support hardware         | MLU270 <br> MLU290 <br>MLU370|
+|        2       |          job type          |    block <br> U1 <br> U4    |
+|        3       |         layout            |  NHWC 、NCHW、ARRAY etc    |
+|        4       |         multidimensional              |       Whether multi-dimensions are supported         |
+|        5       |         zero elements             |       Whether element zero is supported         |
+|        6       |         data type       |         half / float etc           |
+|        7      |        size limit           |       If so, please explain the restrictions and reasons      |
+
+#### 3.1.3 New Feature Testing
+
```
Comment (defei-coder) [inline .github/pull_request_template.md:37]: 应该添加说下面是checklist，希望在PR前完成下面的测试，如果完成标记√

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -7,3 +7,79 @@ Please describe the motivation of this PR and the goal you want to achieve throu
...
+- [ ] Code coverage check, detailed see: [GTest-User-Guide-zh](../docs/GTest-User-Guide-zh.md)
+- [ ] I/O calculation efficiency check, detailed see: [MLU-OPS Performance Acceptance Criteria](../docs/MLU-OPS性能验收标准.md) 
+
+#### 3.1.4 Parameter Check
+
+When a new operator is submitted, the test points are given and the test results are stated.
+
+| Test Point         | Acceptance Criteria | Test result (error message) |
+| -------------- | -------- | -------------------- |
+| Don't conform to the operator restriction | normal error |                      |
```
Comment (defei-coder) [inline .github/pull_request_template.md:57]: 这句得改

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -7,3 +7,79 @@ Please describe the motivation of this PR and the goal you want to achieve throu
...
+|op_name|   |    |     |    |    |    |     |
+
+Platform：MLU370
+
+|operator|mlu_hardware_time(us)|mlu_interface_time(us)|mlu_io_efficiency|mlu_compute_efficiency|mlu_workwpace_size(Bytes)|data_type|shape|
+|-----|----|----|----|----|----|------|-----|
+|op_name|   |    |     |    |    |    |     |
+|op_name|   |    |     |    |    |    |     |
+
+### 3.3 Summary Analysis
```
Comment (defei-coder) [inline .github/pull_request_template.md:85]: 添加类似 ## 2. Modification下的文字描述

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -7,3 +7,79 @@ Please describe the motivation of this PR and the goal you want to achieve throu
...
+## 3. Test Report
+
+If you want to know how to do operator testing, you can see [GTest-User-Guide-zh](../docs/GTest-User-Guide-zh.md)
+
+### 3.1 Modify The Description
+
+#### 3.1.1 Accuracy Acceptance Standard
+
+- Static threshold standard
+  Detailed see: [MLU-OPS Accuracy Acceptance Standard](../docs/MLU-OPS精度验收标准.md)
```
Comment (GraceGuanH) [inline .github/pull_request_template.md:20]: For static threshold standard details, see [MLU-OPS Accuracy Acceptance Standard](../docs/MLU-OPS精度验收标准.md).

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -7,3 +7,79 @@ Please describe the motivation of this PR and the goal you want to achieve throu
...
+- [ ] diff1, diff1 <= 3e-3
+- [ ] diff2, diff2 <= 3e-3
+
+#### 3.1.2 Operator Scheme checklist
```
Comment (GraceGuanH) [inline .github/pull_request_template.md:24]: checklist首字母大写

File: .github/pull_request_template.md [hunk 7]
```diff
@@ -7,3 +7,79 @@ Please describe the motivation of this PR and the goal you want to achieve throu
...
+Platform ：MLU290
+
+|operator|mlu_hardware_time(us)|mlu_interface_time(us)|mlu_io_efficiency|mlu_compute_efficiency|mlu_workwpace_size(Bytes)|data_type|shape|
+|-----|----|----|----|----|----|------|-----|
+|op_name|   |    |     |    |    |    |     |
+|op_name|   |    |     |    |    |    |     |
+
+Platform：MLU370
+
+|operator|mlu_hardware_time(us)|mlu_interface_time(us)|mlu_io_efficiency|mlu_compute_efficiency|mlu_workwpace_size(Bytes)|data_type|shape|
```
Comment (GraceGuanH) [inline .github/pull_request_template.md:80]: operator > Operator

File: .github/pull_request_template.md [hunk 8]
```diff
@@ -7,3 +7,79 @@ Please describe the motivation of this PR and the goal you want to achieve throu
...
+Platform ：MLU270
+
+|operator|mlu_hardware_time(us)|mlu_interface_time(us)|mlu_io_efficiency|mlu_compute_efficiency|mlu_workwpace_size(Bytes)|data_type|shape|
+|-----|----|----|----|----|----|------|-----|
+|op_name|   |    |     |    |    |    |     |
+|op_name|   |    |     |    |    |    |     |
+
+Platform ：MLU290
+
+|operator|mlu_hardware_time(us)|mlu_interface_time(us)|mlu_io_efficiency|mlu_compute_efficiency|mlu_workwpace_size(Bytes)|data_type|shape|
```
Comment (GraceGuanH) [inline .github/pull_request_template.md:73]: Operator

File: .github/pull_request_template.md [hunk 9]
```diff
@@ -7,3 +7,79 @@ Please describe the motivation of this PR and the goal you want to achieve throu
...
+| Don't conform to the operator restriction | normal error |                      |
+|  Illegal parameter passing  | normal error |                      |
+
+### 3.2 Performance Test
+
+Detailed see：[MLU-OPS Performance Acceptance Criteria](../docs/MLU-OPS性能验收标准.md)
+
+Platform ：MLU270
+
+|operator|mlu_hardware_time(us)|mlu_interface_time(us)|mlu_io_efficiency|mlu_compute_efficiency|mlu_workwpace_size(Bytes)|data_type|shape|
```
Comment (GraceGuanH) [inline .github/pull_request_template.md:66]: Operator

File: .github/pull_request_template.md [hunk 10]
```diff
@@ -7,3 +7,79 @@ Please describe the motivation of this PR and the goal you want to achieve throu
...
+When a new operator is submitted, the test points are given and the test results are stated.
+
+| Test Point         | Acceptance Criteria | Test result (error message) |
+| -------------- | -------- | -------------------- |
+| Don't conform to the operator restriction | normal error |                      |
+|  Illegal parameter passing  | normal error |                      |
+
+### 3.2 Performance Test
+
+Detailed see：[MLU-OPS Performance Acceptance Criteria](../docs/MLU-OPS性能验收标准.md)
```
Comment (GraceGuanH) [inline .github/pull_request_template.md:62]: See [MLU-OPS Performance Acceptance Criteria](../docs/MLU-OPS性能验收标准.md) for details.

File: .github/pull_request_template.md [hunk 11]
```diff
@@ -7,3 +7,79 @@ Please describe the motivation of this PR and the goal you want to achieve throu
...
+- [ ] Nan / INF tests 
+- [ ] Memory leak checking, detailed see: [GTest-User-Guide-zh](../docs/GTest-User-Guide-zh.md)
+- [ ] Code coverage check, detailed see: [GTest-User-Guide-zh](../docs/GTest-User-Guide-zh.md)
+- [ ] I/O calculation efficiency check, detailed see: [MLU-OPS Performance Acceptance Criteria](../docs/MLU-OPS性能验收标准.md) 
+
+#### 3.1.4 Parameter Check
+
+When a new operator is submitted, the test points are given and the test results are stated.
+
+| Test Point         | Acceptance Criteria | Test result (error message) |
```
Comment (GraceGuanH) [inline .github/pull_request_template.md:55]: Test Result (Error Message)

File: .github/pull_request_template.md [hunk 12]
```diff
@@ -7,3 +7,79 @@ Please describe the motivation of this PR and the goal you want to achieve throu
...
+#### 3.1.2 Operator Scheme checklist
+
+|      Serial Number     |           Demand            |      Demand For Details       |
+|----------------|---------------------------|---------------------|
+|        1       |          support hardware         | MLU270 <br> MLU290 <br>MLU370|
+|        2       |          job type          |    block <br> U1 <br> U4    |
+|        3       |         layout            |  NHWC 、NCHW、ARRAY etc    |
+|        4       |         multidimensional              |       Whether multi-dimensions are supported         |
+|        5       |         zero elements             |       Whether element zero is supported         |
+|        6       |         data type       |         half / float etc           |
```
Comment (GraceGuanH) [inline .github/pull_request_template.md:33]: 表格的内容回头沟通下

File: .github/pull_request_template.md (hunk 3 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:57]: 已修改

File: .github/pull_request_template.md [hunk 13]
```diff
@@ -7,3 +7,79 @@ Please describe the motivation of this PR and the goal you want to achieve throu
 ## 2. Modification
```
[the remark this answers is not in the document]
Comment (tudejiang79) [reply .github/pull_request_template.md:7]: 已修改

[the remark this answers is not in the document]
Comment (tudejiang79) [reply .github/pull_request_template.md:7]: 已修改

File: .github/pull_request_template.md [hunk 14]
```diff
@@ -7,3 +7,79 @@ Please describe the motivation of this PR and the goal you want to achieve throu
...
 
 Please briefly describe what modification is made in this PR ，and indicate where to modify it.
```
[the remark this answers is not in the document]
Comment (tudejiang79) [reply .github/pull_request_template.md:9]: 已修改

File: .github/pull_request_template.md (hunk 5 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:20]: 已修改

File: .github/pull_request_template.md (hunk 6 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:24]: 已修改

File: .github/pull_request_template.md (hunk 7 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:80]: 已修改

File: .github/pull_request_template.md (hunk 8 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:73]: 已修改

File: .github/pull_request_template.md (hunk 9 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:66]: 已修改

File: .github/pull_request_template.md (hunk 10 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:62]: 已修改

File: .github/pull_request_template.md (hunk 11 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:55]: 已修改

File: .github/pull_request_template.md (hunk 12 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:33]: 已修改

File: .github/pull_request_template.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tudejiang79) [reply .github/pull_request_template.md:15]: 已修改

File: .github/pull_request_template.md [hunk 15]
```diff
@@ -7,3 +7,79 @@ Please describe the motivation of this PR and the goal you want to achieve throu
...
+
+## 3. Test Report
+
+If you want to know how to do operator testing, you can see [GTest-User-Guide-zh](../docs/GTest-User-Guide-zh.md)
```
[the remark this answers is not in the document]
Comment (tudejiang79) [reply .github/pull_request_template.md:13]: 已修改

File: .github/pull_request_template.md (hunk 4 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:85]: 已修改

File: .github/pull_request_template.md (hunk 2 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:37]: 已修改

File: .github/pull_request_template.md (hunk 1 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:15]: 已修改

File: .github/pull_request_template.md [hunk 16]
```diff
@@ -1,9 +1,89 @@
...
 ## 2. Modification
 
-Please briefly describe what modification is made in this PR ，and indicate where to modify it.
+Please briefly describe what modification is made in this pull request, and indicate where to make the modification.
+
+## 3. Test Report
+
+If you want to know how to do operator testing, you can see [GTest-User-Guide-zh](../docs/GTest-User-Guide-zh.md).
+
+### 3.1 The Description of modification
```
Comment (GraceGuanH) [inline .github/pull_request_template.md:15]: Modification Details

File: .github/pull_request_template.md [hunk 17]
```diff
@@ -1,9 +1,89 @@
...
+|        2       |          Job types          |    block <br> U1 <br> U4    |
+|        3       |         Layouts            |  NHWC 、NCHW、ARRAY etc    |
+|        4       |         Whether multi-dimensions are supported              |                |
+|        5       |          Whether element zero is supported             |               |
+|        6       |         Data type(half/float)       |         half / float etc           |
+|        7      |        Whether there is size limit           |             |
+
+#### 3.1.3 New Feature Test
+
+If you have checked the following items, please put a check mark in front of the corresponding items.
```
Comment (GraceGuanH) [inline .github/pull_request_template.md:38]: If you have checked the following items, please check the box of the corresponding item.

File: .github/pull_request_template.md [hunk 18]
```diff
@@ -1,9 +1,89 @@
...
+- [ ] Layout test
+- [ ] Different size/integer remainder end segment/alignment misalignment test
+- [ ] Zero dimensional tensor test/zero element test
+- [ ] stability test
+- [ ] Multiple platform test
+- [ ] Gen_case module test
+- [ ] Nan/INF tests 
+- [ ] For memory leak check details, see[GTest-User-Guide-zh](../docs/GTest-User-Guide-zh.md).
+- [ ] For code coverage check details, see: [GTest-User-Guide-zh](../docs/GTest-User-Guide-zh.md).
+- [ ] For I/O calculation efficiency check details see: [MLU-OPS Performance Acceptance Criteria](../docs/MLU-OPS性能验收标准.md).
```
Comment (GraceGuanH) [inline .github/pull_request_template.md:51]: details后加逗号

File: .github/pull_request_template.md [hunk 19]
```diff
@@ -1,9 +1,89 @@
...
+Platform：MLU370
+
+|Operator|Mlu_hardware_time(us)|Mlu_interface_time(us)|Mlu_io_efficiency|Mlu_compute_efficiency|Mlu_workwpace_size(Bytes)|Data_type|Shape|
+|-----|----|----|----|----|----|------|-----|
+|op_name|   |    |     |    |    |    |     |
+|op_name|   |    |     |    |    |    |     |
+
+### 3.3 Summary Analysis
+
+Please give a brief overview here, if any need to note and summarize the content.
```
Comment (GraceGuanH) [inline .github/pull_request_template.md:89]: Please give a brief overview here, if you want to note and summarize the content.

File: .github/pull_request_template.md [hunk 20]
```diff
@@ -1,9 +1,89 @@
-Thanks for your contribution and we appreciate it a lot. Please state your motivation and modification for this pull_request here.
+Thanks for your contribution and we appreciate it a lot. 
 
 ## 1. Motivation
 
-Please describe the motivation of this PR and the goal you want to achieve through this PR.
+Please describe the motivation of this pull request and the goal you want to achieve through this pull request.
```
Comment (GraceGuanH) [inline .github/pull_request_template.md:5]: Please describe your motivation and ....

File: .github/pull_request_template.md (hunk 16 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:15]: 已修改

File: .github/pull_request_template.md (hunk 17 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:38]: 已修改

File: .github/pull_request_template.md (hunk 18 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:51]: 已修改

File: .github/pull_request_template.md (hunk 19 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:89]: 已修改

File: .github/pull_request_template.md (hunk 20 above)
Comment (tudejiang79) [reply .github/pull_request_template.md:5]: 已修改

**Review by GraceGuanH — APPROVED** [review]

**Review by GraceGuanH — APPROVED** [review]

**Review by GraceGuanH — APPROVED** [review]

**Review by WangYuan-cn — APPROVED** [review]

**Review by defei-coder — APPROVED** [review]

**Pull request merged**